src/pyhf/tensor/tensorflow_backend.py

```diff
@@ -408,13 +408,13 @@ def poisson_logpdf(self, n, lam):
             >>> import pyhf
             >>> pyhf.set_backend("tensorflow")
             >>> t = pyhf.tensorlib.poisson_logpdf(5., 6.)
-            >>> print(t)
-            tf.Tensor(-1.8286943, shape=(), dtype=float32)
+            >>> print(t) # doctest:+ELLIPSIS
```

Comment:
ELLIPSIS needed as Linux and MacOS CI system have different floating point on the TF repr print.